radiopadre/casatable.py

```diff
@@ -7,8 +7,10 @@
 
 import radiopadre
 from radiopadre import casacore_tables
+from radiopadre import settings
 from .filelist import FileList
-from .render import render_table, rich_string, render_status_message, render_error, TransientMessage
+from .render import render_table, rich_string, render_status_message, \
```

Comment:
I think instead of "/" we can use "()" to enclose these multiple imports

Reply:
Like this?
```
from .render import (render_table, rich_string)
```
Well I learned something today. :)

Reply:
Yeah, especially for the multiline ones